code/modules/power/engines/singularity/singularity.dm

```diff
@@ -291,24 +291,20 @@
 
 
 /obj/singularity/proc/eat()
-	for(var/tile in spiral_range_turfs(grav_pull, src))
-		var/turf/T = tile
-		if(!T || !isturf(loc))
+	for(var/turf/tile as anything in RANGE_TURFS(grav_pull, src))
```

Comment:
can you add even more performance by changing the blocks in this macro to use the new syntax?
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/a4922e82-1d7b-45d8-ae05-46c0ddf160d4)
You can see similar changes in a PR from CM like https://github.com/cmss13-devs/cmss13/pull/6265/files

Reply:
Oh I completely missed that we could do that now, that's very good

Reply:
Done! Tested and it looks the same!